tools/tsp-client/src/git.ts

```diff
@@ -2,14 +2,17 @@ import { spawn } from "child_process";
 import { simpleGit } from "simple-git";
 
 export async function getRepoRoot(repoPath: string): Promise<string> {
-  return await simpleGit(repoPath).revparse(["--show-toplevel"]);
+  return simpleGit(repoPath).revparse(["--show-toplevel"]);
```

Comment:
There is also [Octokit](https://www.npmjs.com/package/@octokit/rest) which is github's official JS library, it is very popular. I'm not suggesting to change from simple-git, just wanted to mention it in case you need to consider alternatives in the future.

Reply:
Thanks! I'll look into it if I need it in the future :)